store/fdroid/metadata/com.x8bit.bitwarden.yml

```diff
@@ -7,7 +7,7 @@ SourceCode: https://github.com/bitwarden/mobile
 Changelog: https://github.com/bitwarden/mobile/releases
 WebSite: https://bitwarden.com
 
-AuthorName: 8bit Solutions LLC
+AuthorName: Bitwarden Inc
```

Comment:
🤔  I think it's lacking the period at the end on `Inc.` at least that's how I see it on the Google Play Store.

Reply:
@go12 Should it be `Bitwarden, Inc.` here?